### PR TITLE
Fix error in attach/detach to support person object

### PR DESCRIPTION
**Pull request by junjieteoh**

### CODE REVIEW

File: config/checkstyle/checkstyle.xml [hunk 1]
```diff
@@ -77,7 +77,9 @@
     -->
     <module name="UnusedImports"/>
 
-    <module name="AvoidStarImport"/>
+    <module name="AvoidStarImport">
```
Comment (ddx-510) [inline config/checkstyle/checkstyle.xml:80]: Please do not bypass any checkstyles

File: src/test/java/seedu/address/testutil/TypicalPersons.java [hunk 2]
```diff
@@ -1,5 +1,7 @@
 package seedu.address.testutil;
 
+import static seedu.address.logic.commands.CommandTestUtil.*;
```
Comment (ddx-510) [inline src/test/java/seedu/address/testutil/TypicalPersons.java:3]: Please revert it back to list of imports

**Review by ddx-510 — CHANGES_REQUESTED** [review]

**Review by ddx-510 — APPROVED** [review]
LGTM

**Review by 9teMare — APPROVED** [review]
LGTM

**Pull request merged**